src/views/Graph.vue

```diff
@@ -483,7 +510,20 @@ export default {
       }
       return ret
     },
-    getDotCode (nodeDimensions, nodes, edges) {
+    /**
+     * Get the nodes binned by cycle point
+     *
+     * @param {Object[]} nodes
+     * @returns {{ [dateTime: string]: Object[] nodes }} mapping of node to their cycle point.
```

Comment:
Not quite valid JSDoc syntax I don't think.
```suggestion
     * @returns {{ [dateTime: string]: Object[] }} mapping of cycle points to nodes
```